packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -379,7 +379,7 @@ Future<Null> _buildGradleProjectV2(String gradle, BuildInfo buildInfo, String ta
     if (buildInfo.fileSystemScheme != null)
       command.add('-Pfilesystem-scheme=${buildInfo.fileSystemScheme}');
   }
-  if (buildInfo.preferSharedLibrary && androidSdk.ndkCompiler != null) {
+  if (buildInfo.preferSharedLibrary && androidSdk.ndk != null) {
```

Comment:
Oops -- missed the gradle side yesterday. Once this lands I can make the naming updates.

Reply:
oops, I did not see that it says `preferSharedLibrary` either :) I guess this code is not covered with tets.